### ISSUE TITLE
Adopt Spoon's approach of committing generated sources to the repository

### ISSUE THREAD

**Issue by slarse** [SpoonLabs/sorald#294]
My dev environment (IntelliJ IDEA) started acting up last Friday, crashing left and right. I've lost about half a workday since trying to work around it.

I've also tried switching to a different environment, but it's _really_ hard to get an arbitrary development environment to properly recognize the generated sources. As such, I think we should adopt Spoon's approach of generating sources and then committing them to the repository. It just saves so much headache when stuff like this happens.

**Comment by slarse** [comment]
@khaes-kth Have you experienced any issues with the code generation? What editor/IDE do you use?

**Comment by khaes-kth** [comment]
@slarse I use Intellij. And yes, I had some problems at first. But after using Maven for building, it was fixed. Since I have not updated/changed my Intellij, it seems to work for now.

**Comment by slarse** [comment]
Argh, yesterday I started with IntelliJ crashing, and today I started with a kernel panic. So, yeah, I need an easy-to-setup backup.

I agree that the source generation works fine for IntelliJ, but I tried two other environments (VS Code and Vim with plugins), and couldn't configure it correctly after an hour of trying. So when I've got a chance I'll try to make the source generation more compatible with different dev tools.
